src/docfx/lib/HtmlUtility.cs

```diff
@@ -152,7 +152,7 @@ public static string TransformXref(string html, Func<string, bool, int, (string
                 var xref = HttpUtility.HtmlDecode(node.GetAttributeValue("href", ""));
 
                 var raw = HttpUtility.HtmlDecode(
-                    node.GetAttributeValue("data-raw-html", null) ?? node.GetAttributeValue("data-raw-source", null) ?? "");
+                    node.GetAttributeValue("data-raw-html", null) ?? node.GetAttributeValue("data-raw-source", null) ?? $"<span class=\"xref\">{UrlUtility.SplitUrl(xref).path}</span>");
```

Comment:
Also share this logic with `XrefExtension.cs`, line 27, 37 ?

Reply:
@live1206 , do we need share? seems the logic is not totally same

Reply:
We don't need to, the "data-raw-source" generated from markdig here is always itself, so we don't need add fallback logic there.

Reply:
I see, should HTML encode the content